python/tvm/relax/frontend/torch/fx_translator.py

```diff
@@ -808,6 +808,21 @@ def _interpolate(self, node: fx.node.Node) -> relax.Var:
             )
         )
 
+    def _cross_entropy(self, node: fx.node.Node) -> relax.Expr:
+        preds = self.env[node.args[0]]
+        targets = self.env[node.args[1]]
+        weights = node.kwargs["weight"]
+        if weights is not None:
+            weights = self.env[weights]
+        reduction = node.kwargs["reduction"]
+        ignore_index = node.kwargs["ignore_index"]
+
+        return self.block_builder.emit(
+            relax.op.nn.nll_loss(
+                relax.op.nn.log_softmax(preds), targets, weights, reduction, ignore_index
+            )
+        )
+
     ########## Others ##########
```

Comment:
Maybe we can commit it to unity in another PR instead
I also have sth to commit to fx_translator :)